io/disk/vfc-tests/virtual_fc.py

```diff
@@ -357,13 +395,24 @@ def vfchost_define_undefine(self, operation, vfc_dic):
         if output.exit_status != 0:
             self.log.debug(output.stderr)
             self.fail("vfchost %s operation failed" % operation)
+        else:
+            self.log.info("%s of %s succes" % (operation, vfc_dic["vfchost"]))
+        time.sleep(5)
+        self.log.info("verifying %s status after %s operation.... \
+                       " % (vfc_dic["vfchost"], operation))
         self.vfchost_config_status_verify(operation, vfc_dic["vfchost"],
                                           vfc_dic["vios"])
+        self.log.info("verifying %s status after %s operation.... \
+                       " % (vfc_dic["vfc_client"], operation))
         self.vfc_client_status_verify(operation, vfc_dic["vfc_client"])
+        time.sleep(8)
```

Comment:
Can the static sleep times be changed into something dynamic based on some outcome ?

Reply:
we will take a call on this in future.

Reply:
Can a comment be added to change this in future ?